tests/models/onnxModels/exp.onnxtxt

```diff
@@ -0,0 +1,58 @@
+ir_version: 5
+producer_name: "onnx-example"
+graph {
+  node {
+    input: "data"
+    output: "result"
+    op_type: "Exp"
+  }
+ name: "test-exp"
+  input {
+    name: "data"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
```

Comment:
it's up to you, but for simple test you could use simpler share (less chars to type ;))

Reply:
not changing it would be even less chars to type ;-)